portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,87 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Lovemore's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <div class="bg-image img1">
+    <h1>My Portfolio</h1>
+    <p>
+        Hello, I’m Lovemore! I am currently enrolled 
+        at Amherst College. I have become adept at 
+        understanding and utilizing Java, data structures, 
+        object-oriented programming, data mining and data science.
+    </p>
+      </div>
+    <div class="bg-image img2">
+        <h2>Projects</h2>
+        <p>
+            <h3>Beehive Game</h3>
+            <p>
```

Comment:
consider adding some CSS to add some padding and some background to your paragraphs to make them a bit more readable.